keyboards/handwired/justageng/office60/readme.md

```diff
@@ -0,0 +1,126 @@
+# Office60
+
+*A short description of the keyboard/project*
```

Comment:
This needs to be fixed

Reply:
I've replaced the line with a description of the keyboard.